lib/jekyll/filters.rb

```diff
@@ -222,6 +222,9 @@ def where(input, property, value)
     #
     # Returns the filtered array of objects
     def sort(input, property = nil, nils = "first")
+      if input.nil?
+          raise ArgumentError.new("Sort: cannot sort a null object.")
```

Comment:
don't you think `cannot sort a null object` tells you that you're in the `sort` function?